packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -735,7 +735,7 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({
             id: 'copyFilePath'
         }, {
-            execute: () => commands.executeCommand(CommonCommands.COPY_PATH.id)
+            execute: () => commands.executeCommand(CommonCommands.COPY_PATH.id, [])
```

Comment:
If I typed `COPY_PATH` properly, then this is the first potential bug caught by this change.

Reply:
I don't believe this should be called with any arguments. `UriAwareCommandHandlers` (multi, in this case), can be called with an array of URI's or with _anything else_, and in the case of anything else, the selection service will be used to generate the URI list. In this case, calling the command with an empty array will result in nothing ever being copied to the clipboard, because an empty array passes this test:

https://github.com/eclipse-theia/theia/blob/c26f12e54ccb5c860e8b6e3d0810ddd40caad6e2/packages/core/src/common/uri-command-handler.ts#L62

so the empty array will be passed to the handler, and nothing will happen.